zeppelin-web/src/components/repository-create/repository-dialog.html

```diff
@@ -28,7 +28,7 @@ <h4 class="modal-title">Add New Repository</h4>
             <div class="form-group">
               <label class="control-label col-sm-2" for="repoId">ID</label>
               <div class="col-sm-10">
-                <input type="text" class="form-control" id="repoId" ng-model="newRepoSetting.id"
+                <input type="text" pattern="^[a-zA-Z0-9]*$" class="form-control" id="repoId" ng-model="newRepoSetting.id"
```

Comment:
@felixcheung Thank you for reviewing! 
`repository id` will be the path of the RestAPI URL so how about adding `=-.-:` characters?